src/latex/agda-latex-macros.sty

```diff
@@ -446,7 +446,9 @@
 \newcommand{\RTC}{\AgdaFunction{ReflexiveTransitiveClosure}\xspace}
 \newcommand{\RTCI}{\AgdaOperator{\AgdaBound{\AgdaUnderscore{}⊢\AgdaUnderscore{}⇀⟦\AgdaUnderscore{}⟧*\AgdaUnderscore{}}}}
 \newcommand{\RTCB}{\AgdaOperator{\AgdaBound{\AgdaUnderscore{}⊢\AgdaUnderscore{}⇀⟦\AgdaUnderscore{}⟧\AgdaUnderscore{}}}}
-\newcommand{\Set}{\AgdaPrimitive{Set}\xspace}
+% \newcommand{\Set}{\AgdaPrimitive{Set}\xspace}
```

Comment:
There's no need keeping this around then

Reply:
I suppose we should keep the following around instead:
```
\newcommand{\Type}{\AgdaPrimitive{Type}\xspace}
```